### PR TITLE
Base address

### PR DESCRIPTION
**Pull request by eggrobin**
Log stacktrace to all log files, and fix rotation threshold when loading from save.

### CODE REVIEW

File: ksp_plugin/interface.cpp [hunk 1]
```diff
@@ -70,6 +75,16 @@ void principia__InitGoogleLogging() {
               << " version " << principia::base::kCompilerVersion
               << " for " << principia::base::kOperatingSystem
               << " " << principia::base::kArchitecture;
+#if OS_WIN
+   MODULEINFO module_info;
+   memset(&module_info, 0, sizeof(module_info));
+   CHECK(
+      GetModuleInformation(GetCurrentProcess(),
```
Comment (pleroy) [inline ksp_plugin/interface.cpp]: This could probably fit on the previous line.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -451,6 +454,9 @@ where is_in_physics_bubble(vessel)
     ToggleableSection(name   : "Logging Settings",
                       show   : ref show_logging_settings_,
                       render : LoggingSettings);
+    ToggleableSection(name   : "CRASH",
+                      show   : ref show_crash_options_,
+                      render : CrashOptions);
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Are you sure you want this in your release?

File: ksp_plugin/interface.cpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/interface.cpp]: Doen.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

**Pull request merged**